dracut/30ignition/ignition-reboot.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition (reboot)
+DefaultDependencies=false
+Before=ignition-complete.target
+
+# Make sure ExecStart= runs before we switch root
+Before=initrd-switch-root.target
+
+# Make sure root filesystem is mounted
+After=ignition-mount.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+EnvironmentFile=/run/ignition.env
```

Comment:
Do we need this?

Reply:
I'm not sure about this, so I left it untouched, comparing to other service files..

Reply:
The env file is useful if you have code dependent on a specific `PLATFORM_ID`. Otherwise, we don't strictly need it (but it doesn't hurt either of course!).

Reply:
I'd prefer to drop it if we don't need it. Less to confuse people.

Reply:
Agreed, dropping this line